src/gui/Application.cpp

```diff
@@ -127,6 +128,12 @@ Application::Application(int& argc, char** argv)
         qWarning()
             << QObject::tr("The lock file could not be created. Single-instance mode disabled.").toUtf8().constData();
     }
+
+    connect(osUtils, &OSUtilsBase::interfaceThemeChanged, [this]() {
```

Comment:
Even though application object lives throughout the application life cycle, I prefer the connect with context.

```suggestion
    connect(osUtils, &OSUtilsBase::interfaceThemeChanged, this, [this]() {
```